nucypher/cli/painting/help.py

```diff
@@ -35,6 +36,12 @@ def echo_solidity_version(ctx, param, value):
     ctx.exit()
 
 
+def echo_config_location(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+    click.secho(f"{DEFAULT_CONFIG_ROOT}\n{USER_LOG_DIR}")
```

Comment:
Should we label the output? eg.
'Configuration Root: <>'
'Logging Directory: <>'

Reply:
Is that a comment or a question? 😊

There is a part of me that thinks this command will be used with bash standard output redirection.

Reply:
Both... 🤔  😅 

> There is a part of me that thinks this command will be used with bash standard output redirection
Interesting - what's the use case there?

Reply:
Imagine something like `tail -f $(nucypher --log-path)`.
That requires however some modifications to current approach:
* Separate flags for the two current paths (e.g., `--config-root` for config root, `--log-path` for logs).
* Point to files, not directories
* Remove labels

Reply:
So the use case is potentially usage as part of a script - that makes sense

> Point to files, not directories
Is pointing to files too specific? - if the goal is to use the config output information for some other more involved processing task

For example: 
* get the config root, and the do some processing on `ursula.json` or `stakeholder.json` to verify some setting, or to backup nucypher data eg. `ursula.db`
OR
* get the log-path, and search for some log entry in any of the log files including older rolled over files